server/views/submit.ejs

```diff
@@ -1,17 +1,17 @@
 <%var title = 'Submit'%>
 
-<link rel="stylesheet" type="text/css" href="./css/submit.css">
+<!-- <link rel="stylesheet" type="text/css" href="./css/submit.css"> -->

```

Comment:
You cannot just comment out the existing css. Did you not test the page to see what happens when you submit the form?